src/ElmHtml/Query.elm

```diff
@@ -244,6 +253,11 @@ hasBoolAttribute attribute value facts =
             False
 
 
+hasEventListener : String -> Facts msg -> Bool
+hasEventListener eventName facts =
+    Dict.member eventName facts.events
```

Comment:
🎨 `hasEventListener` to me implies that it would be true if facts.events is not empty.